src/toil/cwl/cwltoil.py

```diff
@@ -579,7 +584,7 @@ def visit(
                 resolved, tgt, "WritableDirectory" if copy else "Directory", staged
             )
 
-            if location.startswith("file://") and not self.stage_listing:
+            if location.startswith("file://"):  # and not self.stage_listing
```

Comment:
I take it this condition shouldn't be here anymore?

Reply:
Yep, changing.  Thanks for the catch!